cetmix_tower_server/views/cx_tower_file_view.xml

```diff
@@ -68,6 +68,7 @@
                                 name="keep_when_deleted"
                                 attrs="{'invisible': [('source', '!=', 'tower')]}"
                             />
+                            <field name="secret_ids" widget="many2many_tags" />
```

Comment:
Please make this field visible only to the members of the "Manager" and "Root" groups.

Reply:
ok, but groups are already defined at the model level. Isn't that enough?

Reply:
We need to restrict visibility, not access.